pkg/virt-launcher/notify-client/client.go

```diff
@@ -210,10 +210,7 @@ func (n *Notifier) SendDomainEvent(event watch.Event) error {
 			return false, nil
 		}
 
-		ctx, cancel := context.WithTimeout(context.Background(), n.sendTimeout)
-		defer cancel()
```

Comment:
I may have missed prev threads about this one.
Can you explain why this was removed? I did not expect a timeout to be dropped in favor of the poll one.

I would have expected this:
```go
ctx, cancel := context.WithTimeout(ctx, n.sendTimeout)
defer cancel()
```

Reply:
It was an oversight on my part. The intention was not to remove the timeout. I’ve updated it.